grunt/tasks/check-json.js

```diff
@@ -5,12 +5,16 @@ module.exports = function(grunt) {
 
           var _ = require('underscore');
           var chalk = require('chalk'); // for some nice colouring
+          var path = require("path");
 
           var listOfCourseFiles = ['course', 'contentObjects', 'articles', 'blocks', 'components'];
           var listOfObjectTypes = ['course', 'menu', 'page', 'article', 'block', 'component' ];
 
+          var sourcedir = grunt.config('sourcedir');
```

Comment:
Please can you update as per https://github.com/adaptlearning/adapt_framework/pull/1779#discussion_r142157671?